source/agora/node/Ledger.d

```diff
@@ -1130,11 +1236,15 @@ unittest
 
     auto gen_key_pair = getGenesisKeyPair();
 
-    auto storage = new MemBlockStorage();
+    auto FreezingPeriod = 2016;
+    auto validator_cycle = FreezingPeriod / 2;
+    auto params = new immutable(ConsensusParams)(validator_cycle);
+    // Generates a genesis block containing the validator's enrollment data.
+    const genesis = makeGenesisBlock(params);
+    scope storage = new MemBlockStorage([genesis]);
```

Comment:
Shouldn't this have been resolved by https://github.com/bpfkorea/agora/issues/907? Then we wouldn't have to fix any of this code, because it could just use the existing genesis block enrollments.

Reply:
My comment was on the wrong line. I meant the previous unittests in this modul.e

However since #907 will not be fixed soon, we can keep it like this for now.